mingw-w64-wabt/PKGBUILD

```
@@ -0,0 +1,70 @@
+_realname=wabt
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.0.36
+pkgrel=1
+pkgdesc="The WebAssembly Binary Toolkit is a suite of tools for WebAssembly (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url="https://github.com/WebAssembly/wabt"
+msys2_references=(
+  'archlinux: wabt'
+)
+license=('APACHE-2.0')
+depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
+         "${MINGW_PACKAGE_PREFIX}-openssl")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-cc"
+             'git'
+             "${MINGW_PACKAGE_PREFIX}-python-ply"
+             "${MINGW_PACKAGE_PREFIX}-gtest")
+source=("git+$url#tag=$pkgver"
+        "git+https://github.com/WebAssembly/testsuite"
+        "git+https://github.com/google/googletest"
+        "git+https://github.com/dabeaz/ply")
+validpgpkeys=('5DE3E0509C47EA3CF04A42D34AEE18F83AFDEB23')
+sha512sums=('2c7a31932a9a30614f0ceedb136dfc87f4399d7f80b2a7d5a9dfcd19d5efa9aff3dd5fb469661d309ae32426b2232b5d0799b44d7301758d9660e635bcfe5d66'
+            'SKIP'
+            'SKIP'
+            'SKIP')
+
+prepare() {
+  cd "${srcdir}/${_realname}"
+
+  git submodule init
+  git config submodule."external/testsuite".url "${srcdir}/testsuite"
+  git config submodule."external/googletest".url "${srcdir}/googletest"
+  git config submodule."external/ply".url "${srcdir}/ply"
+  git submodule update --recursive
+}
+
+build() {
+  declare -a extra_config
+  if check_option "debug" "n"; then
+    extra_config+=("-DCMAKE_BUILD_TYPE=Release")
+  else
+    extra_config+=("-DCMAKE_BUILD_TYPE=Debug")
+  fi
+
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+    cmake \
+      -GNinja \
+      -DCMAKE_INSTALL_PREFIX="${MINGW_PREFIX}" \
+      "${extra_config[@]}" \
+      -DBUILD_{SHARED,STATIC}_LIBS=ON \
```

```suggestion
      -DBUILD_SHARED_LIBS=ON \
```
there is no separate option for static libs